src/proof/alethe/alethe_post_processor.cpp

```diff
@@ -1826,6 +1826,68 @@ bool AletheProofPostprocessCallback::update(Node res,
   }
 }
 
+bool AletheProofPostprocessCallback::maybeReplacePremiseProof(Node premise,
```

Comment:
It would be nice to have a comment with a proof tree here as it was done for the proof rules.